app/lib/stellar/server_request_utils.ts

```diff
@@ -276,6 +279,34 @@ const trades = (
     )
 }
 
+const liquidityPools = (
+  server: HorizonServer,
+  {
+    id,
+    accountId,
+    assets,
+    cursor,
+    order = 'asc',
```

Comment:
[note]

It doesn't matter 'asc' or 'desc' much as I think we'll change the order later based on asset locked.
But it seems we see more familiar asset codes on the first view with 'asc' (but at least only now?), so I changed it to asc.

Reply:
`asc` sounds good for now